src/Config/Schema.php

```diff
@@ -284,6 +285,15 @@ public function getSchema()
                     StageConfigInterface::STAGE_GLOBAL => '',
                 ],
             ],
+            BuildInterface::VAR_ERROR_REPORT_DIR_NESTING_LEVEL => [
+                self::SCHEMA_TYPE => ['integer'],
+                self::SCHEMA_VALUE_VALIDATION => range(0, 32),
+                self::SCHEMA_STAGE => [
+                    StageConfigInterface::STAGE_GLOBAL,
+                    StageConfigInterface::STAGE_BUILD,
+                ],
+                self::SCHEMA_DEFAULT_VALUE => [StageConfigInterface::STAGE_BUILD => 1]
```

Comment:
@YPyltiai I am not sure that it is correct to change the report creation for everyone. It looks like BIC

Reply:
@NadiyaS it will go into minor ECE-Tools version (2002.1.0), so should be OK given how much issues it solves in general.

Reply:
@YPyltiai if customer updates `metapackage` (not `ece-tools`) as we recommend, then ece-tools also be updated to the latest minor version

Reply:
@NadiyaS that should be OK in this case anyway - problem with this task is that site can go down completely if this is not fixed. Given you have a variable to disable this behavior (new one), we should be OK to proceed, since otherwise risks are higher and sites will continue going down because of incorrect report generation logic.

I think @kandy was also OK with this approach, since we don't have other options to fix it really.